libs/hdf-converters/src/ckl-mapper/jsonixMapping.ts

```diff
@@ -0,0 +1,587 @@
+/**
+ * Mapping from checklist XSD schema to JS object
+ * (Generated by jsonix-schema-compiler)
```

Comment:
run that cli tool again to validate these results

write up a wiki page explaining the process of going from xml schema to hdf mapper (with all of the intermediate bits explained like running this cli, generating the types, etc)

might need to extend the 'how to write a mapper' wiki page as well or link to your new one from that one

Reply:
Documentation added https://github.com/mitre/heimdall2/wiki/How-to-create-and-use-a-JSONIX-mapping-object